### PR TITLE
Add OncoKB

### PR DESCRIPTION
**Pull request by leexgh**
try 
7:g.55249071C>T (sensitive and resistant)
1:g.115256529T>C (long name)
17:g.41226411G>A (sensitive with multi levels)

image

### CODE REVIEW

File: src/component/variantPage/biologicalFunction/Oncokb.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    [MUTATION_EFFECT.INCONCLUSIVE]: 'inconclusive',
+    [MUTATION_EFFECT.UNKNOWN]: 'unknown',
+};
+
+@observer
+class Oncokb extends React.Component<IOncokbProps> {
+    static ONCOKB_URL: string = 'https://oncokb.org/';
+
+    public oncogenicity(oncokb: IndicatorQueryResp) {
+        // TODO do we need to check the ""
```
Comment (onursumer) [inline src/component/variantPage/biologicalFunction/Oncokb.tsx]: Is this `TODO` still valid or does it refer to `oncokb.oncogenic !== ''`?

Comment (leexgh) [reply src/component/variantPage/biologicalFunction/Oncokb.tsx]: @onursumer TODO here is to remind me to double-check if the value could be "" and to see do we need the `oncokb.oncogenic !== ''`. The answer is it's possible to be "", so I should probably remove the TODO now

Comment (leexgh) [reply src/component/variantPage/biologicalFunction/Oncokb.tsx]: @onursumer done

**Comment by leexgh** [comment]
@inodb good idea, will do

**Pull request merged**